sdk/src/test/java/com/mapbox/maps/attribution/AttributionParseTest.java

```diff
@@ -310,15 +312,40 @@ public void testShortOpenStreetMapWithoutCopyrightString() throws Exception {
   @Test
   public void testWithImproveThisMapString() throws Exception {
     AttributionParser attributionParser = new AttributionParser.Options(RuntimeEnvironment.application)
-      .withAttributionData(STREETS_ATTRIBUTION, "blabla", "")
+      .withAttributionData(STREETS_ATTRIBUTION, CUSTOM_ATTRIBUTION, "")
       .withImproveMap(true)
       .withCopyrightSign(false)
       .build();
 
     assertEquals(
       "Attribution string should match",
-      "© Mapbox / OSM / Improve This Map",
+      "© Mapbox / OSM / Improve This Map / Custom Contributors",
       attributionParser.createAttributionString(true)
     );
   }
+
+  @Test
+  public void testParseAttributionWithCustomStringAttribution() throws Exception {
```

Comment:
pretty unexpected to see test in Java here 😄

Reply:
We need to convert it to kotlin, probably later :D